README.md

```diff
@@ -1,81 +1,127 @@
-# Vera Android SDK 
```

Comment:
I'm missing a short overview on the SDK, similar to what Alex added to iOs:

The VeraSDK provides access to the Vera platform to any Native application. When the user finds themselves in any of the Vera supported sites (buildings), they can open Vera, and it will localize them inside the building with very accurate precision. Once localized, the user can access any AR Experiences (ARXs) set up for that specific site.